src/linear_algebra/basis.lean

```diff
@@ -341,6 +342,33 @@ lemma is_basis.smul_eq_zero [no_zero_divisors R] {c : R} {x : M} :
 
 end
 
+lemma is_basis.smul_is_basis {v : ι → M} (hv : is_basis R v)
```

Comment:
We already have [`is_basis.smul`](https://leanprover-community.github.io/mathlib_docs/ring_theory/algebra_tower.html#is_basis.smul) - so maybe this should be called `is_basis.smul_of_invertible`